### DISCUSSION TITLE
Any way to have multiple mod themes within the same mod?

### DISCUSSION THREAD

**Discussion by here2reportissues** [ramensoftware/windhawk-mods#922] [Q&A]
I want to have the no recommended section, side by side #2, and translucent themes all at the same time. Is this possible?

**Comment by m417z** [comment]
It's possible, but requires some manual work of merging the styles manually. The theme pages here can help:
https://github.com/ramensoftware/windows-11-start-menu-styling-guide

**Reply by here2reportissues** [reply]
I'm dumb as hell when it comes to this. Can you please ELI5? lol

**Reply by m417z** [reply]
I don't think a 5 year old can do this. Start by following the link and looking at the examples, then look at the themes and which styles are used for them. Then you'll naturally figure out how to merge the three themes.
